ofl/yarndings12/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,3 @@
+<p>The Soft Type Collection is designed for knitters to chart out their typographic projects. Yarndings is an ornamental typeface inspired by a mix of typographic ornamentation, traditional Fair Isle & Nordic knitting patterns, and dingbat fonts from the 1990's.</p>
```

Comment:
The description file still needs to include the note about numbers in the File name + the link to the charted version.